<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9841J3A" failures="1" tests="1" name="Test" time="24.33" errors="0" timestamp="2023-10-26T20:15:07 IST">
  <testcase classname="com.fastparking.testcases.TC_ParkingOwners" name="verifyAddParkingOwner" time="15.85">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;...wner added successfully!&amp;gt; but was:&amp;lt;...WNER_ALREADY_EXIST&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<...wner added successfully!> but was:<...WNER_ALREADY_EXIST>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at com.fastparking.testcases.TC_ParkingOwners.verifyAddParkingOwner(TC_ParkingOwners.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddParkingOwner -->
</testsuite> <!-- Test -->
